### ISSUE TITLE
No onyx/nef/rend buffs

### ISSUE THREAD

**Issue by animism** [LightsHope/server#769]
Hi

My character can't seem to receive Onyxia, Nefarian OR Rend buffs.
It's only 1 of my characters that this doesn't work on, but it is my main character and it makes it very hard to raid efficiently when my guild spends so much time getting everyone buffs and I get nothing from it.
The character name is Nochanice - Horde.
I tried removing my WTF and WTB folder, removing addons and logging in, but it didn't work.

**Comment by LEGAZIZ** [comment]
what is your reputation with orgrimmar?

**Comment by fgsfdss** [comment]
I don't think that should matter.

**Comment by Nelythia** [comment]
It does. You need honored with orgrimmar or stormwind to receive buffs

**Comment by fgsfdss** [comment]
I looked up some information and could not see that mentioned anywhere, except this comment on wowhead for WCB where it was even disproved. http://www.wowhead.com/spell=16609/warchiefs-blessing#comments:id=152779

I also got it numerous times on my bank alts that are all friendly.

**Comment by berillis** [comment]
Yes, you need friendly status with Orgrimmar to receive the onyxia head buff.
